src/common/scheduling.js

```diff
@@ -289,7 +294,8 @@ const valueToCron = value => {
   onChange: propTypes.func.isRequired,
   range: propTypes.array,
   labelId: propTypes.string.isRequired,
-  value: propTypes.any.isRequired
+  value: propTypes.any.isRequired,
+  weekDayMode: propTypes.bool
 })
 class TimePicker extends Component {
```

Comment:
I would prefer if `TimePicker` stay as generic as possible and you create a wrapper component for days `DayPicker`.